frontend/packages/kubevirt-plugin/src/constants/vm/vm-status.ts

```diff
@@ -54,25 +54,22 @@ export class VMStatus extends StatusEnum<VMStatusSimpleLabel | StatusSimpleLabel
     'Launcher pod error',
     { isError: true },
   );
-  static readonly CDI_IMPORT_ERROR = new VMStatus('VMStatus_CDI_IMPORT_ERROR', 'Import error', {
+  static readonly CDI_IMPORT_ERROR = new VMStatus('VMStatus_CDI_IMPORT_ERROR', 'Disk image error', {
     isError: true,
-    group: StatusGroup.CDI,
   });
   static readonly CDI_IMPORTING = new VMStatus(
     'VMStatus_CDI_IMPORTING',
-    StatusSimpleLabel.Importing,
+    StatusSimpleLabel.Copying,
```

Comment:
This will cause the filtering to break as it will no longer be possible to filter it by `Importing` filter.